src/internal/modules/invoice-accounts/pre-handlers.js

```diff
@@ -2,9 +2,32 @@
 
 const Boom = require('boom');
 const { partial } = require('lodash');
-
+const dataService = require('./services/data-service');
+const { has } = require('lodash');
+const helpers = require('./lib/helpers');
 const { water } = require('../../lib/connectors/services');
 
+const loadCompanies = async (request) => {
```

Comment:
The loadCompanies, loadBillingContact and getCompany methods all get the similar data. If you stick with the one getCompany and only get the agent company when there is an agent it will be less code and avoid duplication.

Reply:
Fair point. I have removed the unnecessary prehandlers.